strongbox-security/strongbox-authentication-registry/pom.xml

```diff
@@ -81,14 +81,8 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.directory.server</groupId>
-            <artifactId>apacheds-core</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.directory.server</groupId>
-            <artifactId>apacheds-protocol-ldap</artifactId>
-            <scope>test</scope>
+            <groupId>com.unboundid</groupId>
+			<artifactId>unboundid-ldapsdk</artifactId>
         </dependency>
```

Comment:
Cosmetic: Incorrectly indented.

Reply:
I am unsure why it looks like this, it looks fine on my end but I tried to fix it.